pages/documentation/developer/tutorials/sensitivity-analysis.md

```diff
@@ -68,8 +68,8 @@ Create a new Maven's `pom.xml` file in `sensitivity/initial` with the following
     <properties>
         <maven.exec.version>1.6.0</maven.exec.version>
         <slf4j.version>1.7.22</slf4j.version>
-        <powsybl.core.version>3.3.0</powsybl.core.version>
-        <powsybl.rte-core.version>2.10.0</powsybl.rte-core.version>
+        <powsybl.core.version>3.5.0</powsybl.core.version>
+        <powsybl.rte-core.version>2.12.0</powsybl.rte-core.version>
```

Comment:
We should update the version to the next powsybl version (3.7.0 and 2.14.0), because the new API won't be available in powsybl 3.5.0